### ISSUE TITLE
Set perPixelLighting to true by default

### ISSUE THREAD

**Issue by stalgiag** [processing/p5.js#4161]
#### Nature of issue?

- [ ] Found a bug
- [x] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [x] WebGL
- [ ] Other (specify if possible)

Making an intentional duplicate of #3940 because that thread became a little messy. After #4159 , per-pixel lighting is working on all of the most common devices. 

The todo here is to:
> The examples can have the `setAttributes('perPixelLighting', true)` calls removed. Doing that along with changing the [default in this function](https://github.com/processing/p5.js/blob/master/src/webgl/p5.RendererGL.js#L176) and [changing the documentation](https://github.com/processing/p5.js/blob/master/src/webgl/p5.RendererGL.js#L308) for `setAttributes()` to reflect the new default value, should do the job quite nicely!

@LisaMabley do you still have interest in contributing to this issue?
